### ISSUE TITLE
Question about the used optimizer AdamWeightDecayOptimizer

### ISSUE THREAD

**Issue by bzhangGo** [google-research/bert#153]
Perhaps I missed something. But I read from the [optimization](https://github.com/google-research/bert/blob/master/optimization.py#L134) where the author didnot use the bias-corrected momentum for parameter update. In other words, instead of use
m_t = m_t / (1 - beta_1^t),
the author used m_t directly. 

My question is: Is there any reason to abandon this correction? Or this is better for training bert?

**Comment by jacobdevlin-google** [comment]
This was mostly an oversight, but doesn't really matter when using a reasonable learning rate warmup.
